qiskit/qpy/formats.py

```diff
@@ -26,6 +42,23 @@
 FILE_HEADER_PACK = "!6sBBBBQ"
 FILE_HEADER_SIZE = struct.calcsize(FILE_HEADER_PACK)
 
+# CIRCUIT_HEADER_V10
+CIRCUIT_HEADER_V10 = namedtuple(
+    "HEADER",
+    [
+        "name_size",
+        "global_phase_type",
+        "global_phase_size",
+        "num_qubits",
+        "num_clbits",
+        "metadata_size",
+        "num_registers",
+        "num_instructions",
+    ],
+)
+CIRCUIT_HEADER_V10_PACK = "!H1cHIIQIQ?"
+CIRCUIT_HEADER_V10_SIZE = struct.calcsize(CIRCUIT_HEADER_V10_PACK)
+
```

Comment:
Is this used?  I think it's the same as `CIRCUIT_HEADER_V2`.

Reply:
oops, yes, leftover from when I was adding it to the circuit header.